### ISSUE TITLE
Memory availability check

### ISSUE THREAD

**Issue by PaszaVonPomiot** [TryGhost/Ghost-CLI#1185]
### Summary
When trying to update ghost i get: 
`You are recommended to have at least 150 MB of memory available for smooth operation. It looks like you have ~140 MB available`
And it seems it doesn't take into account swap because mine is fully available:
```
$ free -h
              total        used        free      shared  buff/cache   available
Mem:           985M        640M         97M         12M        248M        178M
Swap:          1.9G        188M        1.7G
```

### Steps to Reproduce

- sudo npm install -g npm
- sudo npm install ghost-cli@latest -g
- ghost update

### Log file
```
1) SystemError

Message: You are recommended to have at least 150 MB of memory available for smooth operation. It looks like you have ~140 MB available.

Debug Information:
    OS: Ubuntu, v18.04.3 LTS
    Node Version: v10.17.0
    Ghost Version: 2.28.0
    Ghost-CLI Version: 1.13.1
    Environment: production
    Command: 'ghost update'
```

**Comment by oskarkrawczyk** [comment]
I do wonder if there's a specific reason why only available hardware memory is checked.

`systeminformation` itself provides info about available swap, so there must have been a reason not to utilize that information.

**Comment by acburdine** [comment]
@PaszaVonPomiot @oskarkrawczyk that may have just been an oversight when we added systeminformation - I'll look and see if we can update that check to include free swap as well

**Comment by oskarkrawczyk** [comment]
@acburdine you're the man! Thank you for this!

**Comment by acburdine** [comment]
Fix released in Ghost-CLI 1.14.1
